examples/MNIST_classification.ipynb

```diff
@@ -0,0 +1,636 @@
+{
```

Comment:
<blockquote>the pixel values increase as we move from the upper-right to bottom-left of the image!</blockquote><p>This doesn't sound right to me in this case</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
The quoted text refers to the filtered "8" (screenshot attached). The center is chosen in the upper right quadrant of the image, so (unless I've misunderstood something), the pixel values of the "8" increase as we move to the lower left quadrant because these values are further away from the center.

<img width="370" alt="Screen Shot 2020-09-25 at 7 13 06 pm" src="https://user-images.githubusercontent.com/26859204/94296560-53bd9500-ff63-11ea-8c19-43a139edaebf.png">

Reply:
Ah yes, I guess I was being uber pedantic about the fact that values also increase going down for instance. But it's really ok as it currently is.